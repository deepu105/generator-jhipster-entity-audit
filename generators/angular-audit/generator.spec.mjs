import { before, describe, expect, it } from 'esmocha';

import { helpers, lookups } from '#test-utils';

const SUB_GENERATOR = 'angular-audit';
const SUB_GENERATOR_NAMESPACE = `jhipster-entity-audit:${SUB_GENERATOR}`;

describe('SubGenerator angular-audit of entity-audit JHipster blueprint', () => {
  describe('run', () => {
    let result;
    before(async function () {
      result = await helpers
        .create(SUB_GENERATOR_NAMESPACE)
        .withOptions({
          reproducible: true,
          defaults: true,
          baseName: 'jhipster',
          skipMenu: true,
        })
        .withLookups(lookups)
        .run();
    });

    it('should succeed', () => {
      expect(result.getStateSnapshot()).toMatchSnapshot();
    });
  });
});
