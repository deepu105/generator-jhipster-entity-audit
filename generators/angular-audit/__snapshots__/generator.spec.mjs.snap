// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubGenerator angular-audit of entity-audit JHipster blueprint run should succeed 1`] = `
{
  ".yo-rc.json": {
    "stateCleared": "modified",
  },
  "package.json": {
    "stateCleared": "modified",
  },
  "src/main/webapp/app/admin/entity-audit/entity-audit-event.model.ts": {
    "stateCleared": "modified",
  },
  "src/main/webapp/app/admin/entity-audit/entity-audit-modal.component.html": {
    "stateCleared": "modified",
  },
  "src/main/webapp/app/admin/entity-audit/entity-audit-modal.component.ts": {
    "stateCleared": "modified",
  },
  "src/main/webapp/app/admin/entity-audit/entity-audit-routing.module.ts": {
    "stateCleared": "modified",
  },
  "src/main/webapp/app/admin/entity-audit/entity-audit.component.html": {
    "stateCleared": "modified",
  },
  "src/main/webapp/app/admin/entity-audit/entity-audit.component.ts": {
    "stateCleared": "modified",
  },
  "src/main/webapp/app/admin/entity-audit/entity-audit.module.ts": {
    "stateCleared": "modified",
  },
  "src/main/webapp/app/admin/entity-audit/entity-audit.service.ts": {
    "stateCleared": "modified",
  },
}
`;
